ansible/roles/supervisor/files/etc/supervisor/conf.d/s142.okserver.org/icingaweb2.conf

```diff
@@ -0,0 +1,7 @@
+[program:icingaweb2]
+command=/usr/bin/spawn-fcgi -f /usr/bin/fcgiwrap -F 3 -u www-data -g www-data -a 127.0.0.1 -p 9000 -P /var/run/fastcgi-c.pid
```

Comment:
I would really recommend using `php-fpm` if it's PHP. We already have a role for it.